ansible/roles/pyenv/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: pyenv depencencies present
+  yum:
+    name: ['gcc', 'zlib-devel', 'bzip2', 'bzip2-devel', 'readline-devel', 'sqlite', 'sqlite-devel',
+    'openssl-devel', 'tk-devel', 'libffi-devel']
+    state: present
+
+- name: pyenv root present
+  file:
+    path: '{{ pyenv_root }}'
+    state: directory
+    mode: '0777'
+
+- name: pyenv source cloned
+  tags:
+      - pyenv
+  git:
+    repo: https://github.com/pyenv/pyenv.git
+    dest: '{{ pyenv_root }}'
+    version: eb68ec9488f0df1f668e9272dd5bd8854edf1dff
```

Comment:
Make this an ansible variable and put in the defaults.yml file for the role.

Reply:
Done